deploy_docs.sh

```diff
@@ -5,6 +5,12 @@
 
 GIT_NAME=$(git config --get user.name)
 GIT_EMAIL=$(git config --get user.email)
+GIT_REMOTE_URL=$(git config --get remote.origin.url)
```

Comment:
Not sure this will work on CI. I think CI might be checking out detached HEAD.

Reply:
Actually I can see that I'm just doing `git push` in [another workflow](https://github.com/resoai/TileBoard-addon/blob/b6a67d80c82edb634503110b31de91ed6e1bbb79/.github/workflows/on_tileboard_release.yaml#L37-L37) so it should be working.

The only difference I also have `fetch-depth: 0  # with tags` set for the checkout step.